src/modules/endpoint/sagas/loadLanguages.js

```diff
@@ -6,17 +6,22 @@ import { createLanguagesEndpoint, LanguageModel, Payload } from '@integreat-app/
 import request from '../request'
 import { baseUrl } from '../constants'
 import type { DataContainer } from '../DataContainer'
-import DatabaseContext from '../DatabaseContext'
 
-export default function * loadLanguages (context: DatabaseContext, dataContainer: DataContainer, shouldUpdate: boolean): Saga<void> {
-  const languagesAvailable = yield call({ context: dataContainer, fn: dataContainer.languagesAvailable }, context)
+export default function * loadLanguages (
+  city: string,
+  dataContainer: DataContainer, shouldUpdate: boolean
+): Saga<void> {
+  const languagesAvailable = yield call(
+    (city: string) => dataContainer.languagesAvailable(city),
```

Comment:
Same again

Reply:
only city is required here

Reply:
I meant the ` yield call(dataContainer.categoriesAvailable, city, language)` proposal, but apparently the same answer.